pkgs/by-name/ca/cargo-lambda/package.nix

```diff
@@ -19,10 +17,10 @@ rustPlatform.buildRustPackage rec {
   version = "1.5.0";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-58kVtwBZEAlv9eVesqmWMZ+KxAwEiGMm8mCf9X5tPMI=";
```

Comment:
No idea why the hash in 24.11 differed from the one in master

Reply:
Verified manually, the previous hash was correct, will restore and double check the hash on master

Reply:
Never mind, master is on 1.6.2 and 24.11 on 1.5.0, so this checks out :)